website/docs/r/cluster.html.markdown

```diff
@@ -361,6 +361,28 @@ In addition to all arguments above, the following attributes are exported:
     - DELETED
     - REPAIRING
 
+### Cloud Provider Snapshot Backup Policy
```

Comment:
We shouldn't export this unless here unless they are using cloud_provider_snapshots.  I'm increasingly thinking the compute needs a check for provider_backup_enabled being true since this doesn't apply if backup_enabled = true.

Reply:
Internally if the user no uses the backup_enabled = true, that part of the state will have clean fields until the user enables the backup_enabled, I think the best way is to comment this

Reply:
Even if the user a user enables backup_enabled it will (should) have clean fields as that has nothing to do this cloud provider backup, other than they are mutually exclusive.  I will test what is returned from this resource in the API if a user uses the old backup_enabled.  Hopefully I can still test it as we are deprecating it for new clusters!

Reply:
it is validated for both provider backup enabled and backup enabled attribute for now.

Reply:
Unresolving as I sent some screen shots over slack and want to be sure it is clear that backup_enabled and snapshot backup policy are mutually exclusive as I'm not sure it is.   Only provider_backup_enabled and snapshot_backup_ policy work together.  Let me know if I can answer any more questions on that.

Reply:
I will add a note about your last comment, "Cloud Provider Snapshot Backup Policy will add if provider_backup_enabled is enabled, instead, it will keep empty"

Reply:
Peferct, maybe just a bit clearer with:

Cloud Provider Snapshot Backup Policy will be added if provider_backup_enabled is enabled, if not, returned values will be empty.

Reply:
Thank you I will change it right now